examples/java/src/main/java/org/apache/beam/examples/MinimalWordCount.java

```diff
@@ -112,11 +110,13 @@ public String apply(KV<String, Long> input) {
                          return input.getKey() + ": " + input.getValue();
                        }
                      }))
+
      // Concept #4: Apply a write transform, TextIO.Write, at the end of the pipeline.
      // TextIO.Write writes the contents of a PCollection (in this case, our PCollection of
-     // formatted strings) to a series of text files in Google Cloud Storage.
-     // CHANGE 3/3: The Google Cloud Storage path is required for outputting the results to.
-     .apply(TextIO.Write.to("gs://YOUR_OUTPUT_BUCKET/AND_OUTPUT_PREFIX"));
+     // formatted strings) to a series of text files.
+     //
+     // Replace the string 'YOUR_OUTPUT_FILE' with an actual location on your local disk.
```

Comment:
Given this is direct runner, let's just use the current directory so there are no changes needed.